src/main/java/DrukmakoriSivatag/Plumber.java

```diff
@@ -119,14 +122,14 @@ public boolean pickUpPump(){
      * meghívja az element attribútum Split(Pump p) metódusát. Ezt csak a Pipe implementálja nem
      * üres függvénytörzzsel, ezért, ha más fajta elemen áll a játékos, akkor nem tud lerakni pumpát.
      *
-     * @return  Sikeres volt-e a művelet.
+     * @return Sikeres volt-e a művelet.
      */
     @Override
-    public boolean addPump(){
+    public boolean addPump() {
         boolean success = false;
```

Comment:
The function can be rewritten without this variable the same way as in pickupPump